Decisions.Equifax/PrequalificationOfOne/Response/PreQualificationOfOneResponse.cs

```diff
@@ -0,0 +1,611 @@
+
+            [JsonProperty("inquiries")] public Inquiry[] Inquiries { get; set; }
+
+            [JsonProperty("models")] public Model[] Models { get; set; }
+
+            [JsonProperty("identification")] public Identification Identification { get; set; }
+
+            [JsonProperty("attributes")] public EquifaxUsConsumerCreditReportAttribute[] Attributes { get; set; }
+
+            [JsonProperty("onlineGeoCode")] public OnlineGeoCode[] OnlineGeoCode { get; set; }
+
+            [JsonProperty("OFACAlerts")] public OfacAlert[] OfacAlerts { get; set; }
+
+            [JsonProperty("consumerReferralLocation")]
+            public ConsumerReferralLocation ConsumerReferralLocation { get; set; }
+
+            [JsonProperty("alternateDataSources")] public AlternateDataSources AlternateDataSources { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public class AddressElement
+        {
+            [JsonProperty("addressType")] public string AddressType { get; set; }
+
+            [JsonProperty("houseNumber")] public long HouseNumber { get; set; }
+
+            [JsonProperty("streetName")] public string StreetName { get; set; }
+
+            [JsonProperty("streetType")] public string StreetType { get; set; }
+
+            [JsonProperty("cityName")] public string CityName { get; set; }
+
+            [JsonProperty("stateAbbreviation")] public string StateAbbreviation { get; set; }
+
+            [JsonProperty("zipCode")] public string ZipCode { get; set; }
+
+            [JsonProperty("sourceOfAddress")]
+            [JsonConverter(typeof(FraudVictimIndicatorDataConverter))]
+            public FraudVictimIndicator SourceOfAddress { get; set; }
+
+            [JsonProperty("dateFirstReported")] public string DateFirstReported { get; set; }
+
+            [JsonProperty("dateLastReported")] public string DateLastReported { get; set; }
+
+            [JsonProperty("addressLine1")] public string AddressLine1 { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public class FraudVictimIndicator
+        {
+            [JsonProperty("code")] public string Code { get; set; }
+
+            [JsonProperty("description")] public string Description { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public class AlertContact
+        {
+            [JsonProperty("alertType")] public FraudVictimIndicator AlertType { get; set; }
+
+            [JsonProperty("dateReported")] public string DateReported { get; set; }
+
+            [JsonProperty("effectiveDate")] public string EffectiveDate { get; set; }
+
+            [JsonProperty("telephoneNumbers")] public TelephoneNumberElement[] TelephoneNumbers { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public class TelephoneNumberElement
+        {
+            [JsonProperty("telephoneNumberType")] public FraudVictimIndicator TelephoneNumberType { get; set; }
```

Comment:
FraudVictimIndicator has two properties, Code and Description.

TelephoneNumberType and AlertType(above) deserialize to FraudVictimIndicator?

Reply:
looked this up, this looks like the generator just reused the type it already created.

Can we rename this object to something that makes more sense? 
CodeWithDescription would even be better imo, this type name is visible in the product.

Reply:
Yes, there are multiple types that use it